app/scripts/directives/notifications/notificationDrawerWrapper.js

```diff
@@ -154,11 +154,12 @@
       var formatAndFilterEvents = function(eventMap) {
         var filtered = {};
         ensureProjectGroupExists(filtered, $routeParams.project);
-        _.each(eventMap, function(event) {
+        _.each(eventMap, function(event, eventUID) {
```

Comment:
I'd still rather just use `event.metadata.uid` below instead of relying on the map being keyed by UID. Then if it ever changes later in `eventWatchCallback`, this function won't be affected.

Reply:
I don't think the changes to the notification drawer allow for a nested field.  @jeff-phillips-18 

[Code here](https://github.com/patternfly/angular-patternfly/commit/3faf39f662b51fbce2dfd2b2d6a5c6a080ee8ef8#diff-19e6473ef3a258a5670be9ad1c7b8626R24) looks like it has to be top level on the object:

```JavaScript
ng-repeat="notification in notificationGroup.notifications track by $ctrl.notificationTrackField ? notification[$ctrl.notificationTrackField] || $index : $index"
```
If I'm following what you are saying.